### DISCUSSION TITLE
Anyone using RANDOMBIT and ROSC to get seed for random generator ?

### DISCUSSION THREAD

**Discussion by RadekVoltr** [earlephilhower/arduino-pico#488] [Q&A] [answered]
Need to have random generator and need to change seed on each start. Don't want to use "open analog" as is common on Arduino but rather prefer to use ROSC and RANDOMBIT.

But I am not sure if I can do it/should do it. Is ROSC used (by default) in arduino-pico or it is free to use for RANDOMBIT ?

Did anyone tested it ?

**Comment by RadekVoltr** [comment]
just for record - I want use this for led blinking on robot. No security issues at all :-D Just don't want to see same sequence on each startup

**Comment by earlephilhower** [comment, marked as answer]
The core doesn't use the ROSC, so feel free to use it!

**Reply by RadekVoltr** [reply]
Thanks !
